crates/re_sdk/src/msg_sender.rs

```diff
@@ -93,7 +93,7 @@ impl MsgSender {
         Self {
             entity_path: ent_path.into(),
 
-            timepoint: [(Timeline::log_time(), Time::now().into())].into(),
+            timepoint: TimePoint::default(),
```

Comment:
it looks like right now we're no longer using this timepoint anywhere as we're always seem to use the ones set on the recording (plus the ones the recording generates on adding a row)
I find it still very useful to have this here additionally (also makes logically a lot of sense), but feels like we're missing something in the (code) docs and examples about this 🤔 : `MsgSender::timepoint` doc string doesn't talk about this and we don't seem to have an example now that uses `with_timestamp`

Reply:
We're still using that timestamp the same way we did before, it gets added directly into the `DataRow`s that `MsgSender` generates (`MsgSender::into_rows`).

The only somewhat weird part is that these timestamps get potentially overridden by `RecordingStream` if the internal clock already has values for the same timelines.
The reason this is weird is because `MsgSender` shouldn't exist anymore to begin with, it's just annoying and serves no real purpose anymore... it'll go away with codegen, in the meantime I'll mark `with_timestamp` & friends as `#[doc(hidden)` and update the starting guide.